scripts/docker-run-memory-test.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
```

Comment:
I wonder if we should generalize this a bit and put everything before line 70 into some kind of bash util library. Do we do similar stuff in other /scripts/ files?

Reply:
We should be wasn't sure throwing it into this PR was the right play; was thinking I would tackle afterward.

Reply:
Yep, this part is simply copied from the `test_valgrind.sh` file. Same thing is used in `test_massif.sh`.